python/tank/authentication/ui/login_dialog.py

```diff
@@ -1,20 +1,42 @@
 # -*- coding: utf-8 -*-
+################################################################################
+## Form generated from reading UI file 'login_dialog.ui'
+##
+## Created by: Qt User Interface Compiler version 5.15.2
+##
+## WARNING! All changes made in this file will be lost when recompiling UI file!
+################################################################################
+# from .qt_abstraction import QtCore
```

Comment:
Let's avoid updating this file manually after it was generated by `build_resources.sh`.